recipes/newrelic/apm/php/debian-fpm.yml

```diff
@@ -64,15 +64,15 @@ install:
     verify_continue:
       cmds:
         - |
+          echo -e "{{.YELLOW}}Confirmation:  The following options were selected.  If you wish to change an option, please select 'n' at the prompt and restart the installation."
```

Comment:
Suggestion: change this to `WHITE`. My original intention was for YELLOW to be used when reference process restart or other disruptive system changes only.